src/SwiftUIGlue/SharedCode/Glue.swift

```diff
@@ -112,12 +112,11 @@ public func NSHostingView_rootView<T: View>(root : T) -> NSHostingView<T>
 #endif
 
 #if os(watchOS)
-/* TODO @_silgen_name("swiftui_UIHostingController_rootView")
-public func UIHostingController_rootView<T: View>(root : T) -> UIHostingController<T>
+@_silgen_name("swiftui_WKHostingController_rootView")
+public func WKHostingController_rootView<T: View>() -> WKHostingController<T>
 {
-    return UIHostingController(rootView: root)
+    return WKHostingController()
 }
- */
```

Comment:
Ideally, we'd separate these changes into a separate commit so that this commit can be reverted in the future when we go to fix the shared project issue, but it's not a big deal

Reply:
Sure. Noted.